errata_tool/release.py

```diff
@@ -21,22 +22,51 @@ class ReleaseCreationError(Exception):
 
 
 class Release(ErrataConnector):
+    def _setattr(self):
+        attributes = self.data['attributes']
+        relationships = self.data['relationships']
+        manager = relationships['program_manager']['login_name'] if \
```

Comment:
Can we copy the usual `__getattr__` magic method here for all the attributes (and relationships), and then special-case all the ones that don't fit that pattern? I think that would reduce the copy-and-paste here, and it would also make it easier to support new fields as the ET developers add them.

Looks like "id", "product", "program_manager", "url" and "edit_url" will need to be special-cased... what do you think?